### PR TITLE
Short event life option

### PR DESCRIPTION
**Pull request by cantino**
Replaces #808.  I manually tested the scenario import and export with the code that upgrades the old `keep_events_for` from days to seconds.  Scenarios also now have a `schema_version`.  Successfully imported both an old and new schema export.

@dsander, would you mind giving this a final review and maybe deploy to your setup?

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -580,6 +580,3 @@ DEPENDENCIES
   weibo_2!
   wunderground (~> 1.2.0)
   xmpp4r (~> 0.5.6)
-
-BUNDLED WITH
-   1.10.5
```
Comment (dsander) [inline Gemfile.lock]: This 'feature' of bundler is really annoying, since we already have it in the Gemfile, do you mind updating bundler?

**Comment by dsander** [comment]
Tested it locally and it works fine. LGTM :+1:

File: Gemfile.lock (hunk 1 above)
Comment (cantino) [inline Gemfile.lock]: Agreed.  Will fix.

**Comment by cantino** [comment]
Thanks for testing @dsander!  Rebasing and merging.

**Comment by stvnrlly** [comment]
:+1: Excellent, thanks!

**Pull request merged**